docs/source/release/v3.14.0/indirect_inelastic.rst

```diff
@@ -41,7 +41,6 @@ Improvements
 - Added 'MaxScatterPtAttempts' spinbox to Calculate Monte Carlo Absorption. This sets the maximum number of 
   tries to be made to generate a scattering point.
```

Comment:
Has this file been changed by accident?

Reply:
I was having an issue merging and git seemed to be adamant that I handle a merge conflict, I will re-investigate same with the other files, that the same happened to.